# Generated by Django 5.0.14 on 2025-08-24 20:48

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('user', '0007_historicaluser_failed_login_attempts_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='historicaluser',
            name='celular',
            field=models.CharField(blank=True, max_length=9, null=True, verbose_name='Celular'),
        ),
        migrations.AlterField(
            model_name='historicaluser',
            name='dni',
            field=models.CharField(blank=True, max_length=8, null=True, verbose_name='DNI'),
        ),
        migrations.AlterField(
            model_name='user',
            name='celular',
            field=models.CharField(blank=True, max_length=9, null=True, verbose_name='Celular'),
        ),
        migrations.AlterField(
            model_name='user',
            name='dni',
            field=models.CharField(blank=True, max_length=8, null=True, verbose_name='DNI'),
        ),
    ]
